geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueManagerImpl.java

```diff
@@ -1110,9 +1126,8 @@ private void recoverSingleList(final @NotNull InterestType interestType,
   }
 
   private void recoverCqs(Connection recoveredConnection, boolean isDurable) {
-    Map cqs = getPool().getRITracker().getCqsMap();
-    for (Object o : cqs.entrySet()) {
-      Map.Entry e = (Map.Entry) o;
+    Map<CqQuery, Boolean> cqs = getPool().getRITracker().getCqsMap();
+    for (Map.Entry<CqQuery, Boolean> e : cqs.entrySet()) {
       ClientCQ cqi = (ClientCQ) e.getKey();
```

Comment:
If the key is going to be blindly cast to `ClientCQ` the isn't this really a `Map<ClientCQ, Boolean>`.